Core/src/Material/SurfaceMaterialMapper.cpp

```diff
@@ -231,9 +232,10 @@ void Acts::SurfaceMaterialMapper::mapInteraction(
   std::map<GeometryIdentifier, unsigned int> assignedMaterial;
   using VectorHelpers::makeVector4;
   // Neutral curvilinear parameters
-  NeutralCurvilinearTrackParameters start(makeVector4(mTrack.first.first, 0),
-                                          mTrack.first.second,
-                                          1 / mTrack.first.second.norm());
+  NeutralCurvilinearTrackParameters start(
+      makeVector4(mTrack.first.first, 0), mTrack.first.second,
+      1 / mTrack.first.second.norm(), std::nullopt,
+      NeutralParticleHypothesis::photon());
```

Comment:
Are these actually photons? Might be geantinos I suppose. Unsure if that makes a difference downstream.

Reply:
Right, I see, but I'm wondering if we specifically need to be able to encode Geantinos to hand them over to G4 for the material mapping.

Reply:
For me it depends on if we want to use a Geantino hypothesis during reconstruction or not because currently the `ParticleHypothesis` is only used for reconstruction and I think that is a nice separation we should keep.

If we want to add Geantinos we would need some special PDG values or we need a variant to map this correctly

Reply:
:+1: I added a geantino for this use case